src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -197,6 +197,38 @@
 	padding: 0 22px 0 6px;
 }
 
+#terminal-tabs-plus-button {
+	padding-left: 24px;
+	position : sticky;
+	min-height: 10;
+}
+
+.tabs-widget .terminal-tabs-entry.format-widget-row {
+	text-overflow: clip;
```

Comment:
this doesn't work currently. the ellipsis are still shown (which I don't think we want when minimized)